File: atcoder/370/A.cpp
#include <iostream>
using namespace std;

int main()
{
    int l, r;
    cin >> l >> r;

    if (l == 1 && r != 1)
    {
        cout << "Yes" << endl;
    }
    else if (l != 1 && r == 1)
    {
        cout << "No" << endl;
    }
    else
    {
        cout << "Invalid" << endl;
    }

    return 0;
}

File: atcoder/370/A.py
def main():
    l, r = map(int, input().split())
    if l == 1 and r != 1:
        print("Yes")
    elif l != 1 and r == 1:
        print("No")
    else:
        print("Invalid")
if __name__ == "__main__":
    main()